services/search/testing/test_search_engine.py

```diff
@@ -9,8 +11,12 @@ def test_same_weights_for_all():
     """
     result = {1: 3}
 
-    assert result == SearchEngine().search_query(constants.KEYWORDS_ARTICLE_1,
-                                                 constants.SYNONYMS_ARTICLE_1)
+    with mock.patch('requests.post', return_value=constants.KEYWORDS_DB_MOCK_1) as mocked:
+        assert result == SearchEngine().search_query(constants.KEYWORDS_ARTICLE_1,
+                                                     constants.SYNONYMS_ARTICLE_1)
+        mocked.assert_called_once_with(
+            os.getenv('DB_ENDPOINT'),
```

Comment:
I don't think this makes sense in a test. How will `DB_ENDPOINT` get set for the test? I expect that if I run `pytest test_search_engine.py` it should just work, but including this in your unit test means that I have to set an env variable every time I run unit tests or else the tests will fail.

This means you probably need to refactor `search_query` to not call `os.getenv` directly but rather to import the value from somewhere, and then you can mock that import for your unit tests.